### PR TITLE
- Update R version used by appveyor to 4.3.3

### PR DESCRIPTION
**Pull request by PavelBal**
No idea if it will fix the failing unicode vignette, but lets try

### CODE REVIEW

**Comment by Felixmil** [comment]
What the hell, it worked ?

**Comment by Yuri05** [comment]
<details><summary>I always knew it...</summary>
 = </font> 
</details>

**Comment by Felixmil** [comment]
More like:
 = </font>

**Comment by PavelBal** [comment]
I give up.

**Comment by Yuri05** [comment]
> I give up.

<details><summary>Never ever give up 😄 </summary>

</details> 

For now I would suggest to rewrite the test avoiding the problematic units.
For later: maybe we need to do the same for ² etc. what we did for µ.
R+Unicode+Windows is a nightmare

File: appveyor.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@ version: '{build}' # comment to use specific version and uncomment the next line
 
 # Download script file from GitHub
 init:
+  # Trying to enforce unicode support
+  - ps: Set-WinSystemLocale en-US.UTF-8
+  - ps: Start-Sleep -s 5; Restart-Computer
+  - ps: Start-Sleep -s 5
```
Comment (Yuri05) [inline appveyor.yml:11]: this is pretty insane.
Not sure what will happen on AppVeyor 😆

Comment (PavelBal) [reply appveyor.yml:11]: Boom!

Apparently, this is supposed to work. But setting the locale does not.

**Pull request closed without merging**